.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
I emulated what lint staged was doing in essence here.

Reply:
just wondering why you decided to remove lint-staged (not necessarily against, just wanted to know your reasoning)?

Reply:
I was just leaning on  'one less third party library' for the build since doing it in existing tooling was easy enough.